termwiz/src/tmux_cc/mod.rs

```diff
@@ -76,7 +77,8 @@ pub enum Event {
     },
     Output {
         pane: TmuxPaneId,
-        text: String,
+        bytes: Vec<u8>,
+        bytes_pos: usize,
```

Comment:
it's not clear to me what the purpose is for these `bytes_pos` fields; I think that it needs a better name and probably also some comments to explain it